### PR TITLE
test: Add property tests for PrefixRange and Pagination (#455)

### PR DESCRIPTION
**Pull request by ruhatch**
## Description

Add simple property tests for PrefixRange and Pagination functions

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x] added `!` to the type prefix if API or client breaking change
- [x] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] provided a link to the relevant issue or specification
- [x] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [x] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [x] added a changelog entry to `CHANGELOG.md`
- [x] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [x] updated the relevant documentation or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by cyberbono3 — APPROVED** [review]
lgtm

File: orm/iterator_property_test.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+		if upperLimit == 0 {
+			upperLimit = 1
+		}
+		limit := rapid.Uint64Range(1, upperLimit).Draw(t, "limit").(uint64)
+
+		// Reconstruct the slice from the pages
+		reconstructedGroupMembers := make([]*testdata.GroupMember, 0, len(groupMembers))
+		for offset := uint64(0); offset < uint64(len(groupMembers)); offset += limit {
+			pageRequest := &query.PageRequest{
+				Key:        nil,
```
Comment (blushi) [inline orm/iterator_property_test.go:31]: would that make sense to also test this using `Key`?

**Review by blushi — COMMENTED** [review]
Looks good overall, just a small question.

File: orm/iterator_property_test.go (hunk 1 above)
Comment (ruhatch) [reply orm/iterator_property_test.go:31]: Done https://github.com/regen-network/regen-ledger/pull/493/commits/fc946a0163b27722a9e1c7b9f76dd005ca44d613

**Review by blushi — APPROVED** [review]

**Pull request merged**